reverse_proxy/app/proxy.py

```
@@ -7,27 +8,76 @@
-        writer.close() 
+        writer.close()
+
+async def scheduler_loop(back_writer):
+    """キューから最も優先度が高いデータを取り出し、サーバへ送る"""
+    while True:
+        priority, data, session = await scheduling_queue.get() # キューにデータが入るまで非同期で待機
+        try:
+            print(f"[Scheduler] Sending data: Priority={priority:.6f} from {session.client_info}")
+            back_writer.write(data) # データをバックに書き込む
+            await back_writer.drain() # 書き込みバッファが空になるまで待機
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)